src/Middleware/HttpLogging/src/HttpLoggingMiddleware.cs

```diff
@@ -119,7 +119,7 @@ private async Task InvokeInternal(HttpContext context)
                 }
                 else
                 {
-                    _logger.UnrecognizedMediaType();
```

Comment:
Do we not have a similar log for when the response mediate type is unrecognized?

Reply:
I thought this was only ever called for the request, but I see it's called for the response in `BufferingStream`. Updated